api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,28 +1,3 @@
-/*
```

Comment:
Should the copyright be in place here?

Reply:
Will address properly in a future commit when I have more time to retest. The issue is that make manifests regenerates this code and drops our copyright.